package.json

```diff
@@ -154,10 +155,14 @@
     "puppeteer": "^13.7.0",
```

Comment:
question: do we need switch puppeteer to something like playwright too? 😄

Reply:
No, WebdriverIO does all the browser automation. I wonder if having Puppeteer here is still needed. I will check.